pkg/dao/snapshots_test.go

```diff
@@ -21,6 +23,9 @@ import (
 	"gorm.io/gorm"
 )
 
+const NonRedHatRepoSearch = "to"
+const EmptySearch = ""
```

Comment:
I'd pass these values in your test, rather than making them constants. I don't see them being used elsewhere. What are your thoughts?

Reply:
I prefer to use constants than magic values when I can/when it makes sense. 😅 
For the empty search it could be omitted, but the `NonRedHatRepoSearch` makes sense to me, because just having `"to"` in the main call in test case would be confusing unless you are aware that the generated testing non red hat repos start with that prefix. 😄 (and I know I will forget that by the next time I have to do something with these tests 😂 )

Reply:
I agree it makes sense not use magic values. I'd still lean towards not defining global constants for these values when they're only used in a couple of test cases. 

In the case of the empty search, I think the intention of an empty string is clear, so what about removing that constant entirely?

For the "NonRedHatRepoSearch", since it's just for this one test, what about defining that variable within the test?